pinot-query-planner/src/main/java/org/apache/pinot/query/validate/Validator.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.query.validate;
+
+import org.apache.calcite.rel.type.RelDataTypeFactory;
+import org.apache.calcite.sql.SqlOperatorTable;
+import org.apache.calcite.sql.validate.SqlValidatorCatalogReader;
+import org.apache.calcite.sql.validate.SqlValidatorImpl;
+
+
+/**
+ * The {@code Validator} overwrites Calcite's Validator with Pinot specific logics.
+ */
+public class Validator extends SqlValidatorImpl {
+
+  public Validator(SqlOperatorTable opTab, SqlValidatorCatalogReader catalogReader, RelDataTypeFactory typeFactory) {
+    super(opTab, catalogReader, typeFactory, Config.DEFAULT);
```

Comment:
The current CalciteSqlParser code in Pinot uses BABEL conformance and IIUC it was done during migration from PQL to SQL to relax few things on syntax and semantics. Should we use BABEL here as well instead of DEFAULT ?

Reply:
given PQL is deprecated. i dont think we should use BABEL solely because of this.

Reply:
What I meant to say was after we migrate to SQL, we use BABEL conformance in CalciteSqlParser code.